interpreter/namerd/src/main/scala/io/buoyant/namerd/iface/NamerdInterpreterInitializer.scala

```diff
@@ -48,11 +52,34 @@ case class ClientTlsConfig(commonName: String, caCert: Option[String]) {
   }
 }
 
+case class FailureThresholdConfig(
+  minPeriodMs: Option[Int],
+  threshold: Double,
+  windowSize: Option[Int],
+  closeTimeoutMs: Option[Int]
+) {
+  val defaultConfig = ThresholdConfig()
```

Comment:
Let's explicitly set our own default values instead of inheriting from Finagle.  It's okay if the default values are the same, but this way they won't unexpectedly change out from under us if do a Finagle upgrade.